<launch>
  <node name="rosbag_play" pkg="rosbag" type="play" 
        args="/workspaces/ros-concepts/Complex/costmap_compress_decompress/catkin_ws/src/costmap_1second.bag"/>
  <node name="costmap_compression_node" type="costmap_compression_node" pkg="costmap_compress_decompress" output="screen">
    <param name="compression_algorithm" value="bzip2" />
    <param name="compression_topic" value="/costmap/compressed" />
    <param name="map_topic" value="/costmap/costmap" />
    <param name="compression_identifier" value="1" />
  </node>
  <node name="costmap_decompression_node" type="costmap_decompression_node" pkg="costmap_compress_decompress" output="screen">
    <param name="compression_algorithm" value="bzip2" />
    <param name="compression_topic" value="/costmap/compressed" />
    <param name="decompression_topic" value="/costmap/decompressed" />
    <param name="decompression_identifier" value="1" />
  </node>
</launch>
